_includes/analytics/mouseflow.html

```diff
@@ -0,0 +1,9 @@
+<script type="text/javascript">
+  window._mfq = window._mfq || [];
+  (function() {
+  var mf = document.createElement("script");
+  mf.type = "text/javascript"; mf.defer = true;
+  mf.src = "//cdn.mouseflow.com/projects/e9e8497f-ca8a-426e-b0a1-a78247e85aa4.js";
+  document.getElementsByTagName("head")[0].appendChild(mf);
+  })();
+</script>
```

Comment:
`type="text/javascript"` is redundant nowadays.

To make this script only include if we pass an ID, this could look something like

```suggestion
{% if include.MOUSEFLOW_ID and include.MOUSEFLOW_ID != '' %}
<script>
  window._mfq = window._mfq || [];
  (function() {
  var mf = document.createElement("script");
  mf.type = "text/javascript"; mf.defer = true;
  mf.src = "//cdn.mouseflow.com/projects/e9e8497f-ca8a-426e-b0a1-a78247e85aa4.js";
  document.getElementsByTagName("head")[0].appendChild(mf);
  })();
</script>
{% else %}<!-- No Mouseflow ID configured, Mouseflow is disabled -->{% endif %}
```

Then, in our `_config.yml` we can add a `mouseflow_id` variable, and add an `include` line on our landing page.


Let me make those changes and push it to a branch on my fork